app/src/main/java/com/byoutline/kickmaterial/features/projectlist/MainActivity.kt

```diff
@@ -12,12 +13,19 @@ import com.byoutline.kickmaterial.features.selectcategory.DataManager
 import com.byoutline.kickmaterial.model.Category
 import com.byoutline.kickmaterial.utils.KickMaterialBaseActivity
 import com.byoutline.secretsauce.activities.showFragment
+import dagger.android.AndroidInjector
+import dagger.android.DispatchingAndroidInjector
+import dagger.android.support.HasSupportFragmentInjector
+import javax.inject.Inject
 
 
 /**
  * @author Pawel Karczewski <pawel.karczewski at byoutline.com> on 2015-01-03
  */
-class MainActivity : KickMaterialBaseActivity() {
+class MainActivity : KickMaterialBaseActivity(), HasSupportFragmentInjector {
+    @Inject
+    lateinit var dispatchingFragmentInjector: DispatchingAndroidInjector<Fragment>
+    override fun supportFragmentInjector(): AndroidInjector<Fragment> = dispatchingFragmentInjector
```

Comment:
Example of setting Activity scoped Injector. 
This is needed desipte `MainActivity` not needing any injected classes directly.